changes/921.feature.md

```diff
@@ -0,0 +1 @@
+feat: Gather node statistics from kubernetes with agent backend.
```

Comment:
In the news fragment, we don't add the type prefix as this will be pasted in the section according to the type information given as the file extension (`.feature.md`).